rustfmt.toml

```diff
@@ -1,13 +1,7 @@
 edition = "2021"
 
 # Imports
-imports_granularity = "Crate"
-group_imports = "StdExternalCrate"
 reorder_imports = true
 
 # Consistency
 newline_style = "Unix"
-
-# Format comments
-comment_width = 100
-wrap_comments = true
```

Comment:
We enforce nightly fmt checks in polkadot-sdk and I'd like to keep that here too. 

I think instead we need to modify the `rust-toolchain.toml` to be something like
```toml
[toolchain]
channel = "nightly"
components = ["rustfmt"]
```

Reply:
Aha I will remove this from this PR as it is not related.